### PR TITLE
[DR-1245] - Web App details

### PR DESCRIPTION
**Pull request by nreal**
Hi team
I fixed some issues:
- Change the X for ¨per¨y ¨por¨. https://www.screencast.com/t/AtsXBQL7  
- Change templates for ¨Plantillas¨ in spanish. https://www.screencast.com/t/Za6Jznt1tblG 
- Remove mayus. https://www.screencast.com/t/iXMaVDG6OcP (Debería ser View Profile, SMTP & API Information, Domain manager, My plan, Log out ) 
- Fix green color for plan's box. https://screencast.com/t/U3q4m5GgiEm (Ver color en RelayButtonHover.png) 

Could you review?

### CODE REVIEW

File: src/wwwroot/styles/views/_header.scss [hunk 1]
```diff
@@ -102,7 +102,6 @@ header{
       ul{
         list-style-type: none;
         color: $suit-font-color-lighter;
-        text-transform: uppercase;
```
Comment (andresmoschini) [inline src/wwwroot/styles/views/_header.scss]: Did you check _translation_ files? maybe we hard coded the caps...

**Comment by andresmoschini** [comment]
👍

File: src/wwwroot/styles/views/_header.scss (hunk 1 above)
Comment (nreal) [inline src/wwwroot/styles/views/_header.scss]: No, I'll check

Comment (nreal) [inline src/wwwroot/styles/views/_header.scss]: Sorry, yes I checked, in the translation is lowercase

**Comment by nreal** [comment]
@mvillaseco  I added for the green button de style over #82CEB7

**Pull request merged**